python/cudf/cudf/tests/test_orc.py

```diff
@@ -1715,6 +1700,72 @@ def test_orc_columns_and_index_param(index_obj, index, columns):
         assert_eq(expected, got, check_index_type=True)
 
 
+@pytest.mark.parametrize("index_obj", [None, [10, 11, 12], ["x", "y", "z"]])
+@pytest.mark.parametrize("index", [True, False, None])
+@pytest.mark.parametrize(
+    "columns",
+    [
+        None,
+        [],
+    ],
+)
+def test_orc_columns_and_index_param(index_obj, index, columns):
+    run_orc_columns_and_index_param(index_obj, index, columns)
+
+
+@pytest.mark.parametrize(
+    "columns,index,index_obj",
+    [
+        pytest.param(
+            ["a", "b"],
+            True,
+            None,
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12026"
+            ),
+        ),
+        pytest.param(
+            ["a", "b"],
+            True,
+            [10, 11, 12],
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12026"
+            ),
+        ),
+        pytest.param(
+            ["a", "b"],
+            True,
+            ["x", "y", "z"],
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12026"
+            ),
+        ),
+        (["a", "b"], False, None),
+        (["a", "b"], False, [10, 11, 12]),
+        (["a", "b"], False, ["x", "y", "z"]),
+        (["a", "b"], None, None),
+        pytest.param(
+            ["a", "b"],
+            None,
+            [10, 11, 12],
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12026"
+            ),
+        ),
+        pytest.param(
+            ["a", "b"],
+            None,
+            ["x", "y", "z"],
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12026"
+            ),
+        ),
+    ],
+)
```

Comment:
Can we apply mark.xfail to the whole test to simplify the parametrization to not need `pytest.param`?

Reply:
Only some of these parameters fail, but I should split out the ones that don't...

Reply:
Oh I think the GH GUI just wasn't quite readable enough for me to notice the ones that didn't fail or I wouldn't have asked. Thanks for splitting though!